packages/inventory/src/EntityTable.js

```diff
@@ -50,65 +52,55 @@ class EntityTable extends React.Component {
     }
 
     renderCol = (col, key, composed, isTime) => {
-        if (!col.hasOwnProperty('isOpen')) {
-            if (composed) {
-                return (
-                    <div className="ins-composed-col">
-                        { composed.map(path => (
-                            <a key={ path }
-                                widget="col"
-                                data-key={ path }
-                                href={ `${location.pathname}/${col.id}` }
-                                onClick={ event => {
-                                    event.preventDefault();
-                                    event.stopPropagation();
-                                    this.onRowClick(event, col.id);
-                                }}
-                            >
-                                { get(col, path, ' ') || '\u00A0' }
-                            </a>
-                        )) }
-                    </div>
-                );
-            }
-
-            if (isTime) {
-                if (DateFormat) {
-                    return { title: <DateFormat date={get(col, key, ' ')} type="relative" /> };
+        if (composed) {
+            return (
+                { title: <div className="ins-composed-col">
+                    { composed.map(path => (
+                        <a key={ path }
+                            widget="col"
+                            data-key={ path }
+                            href={ `${location.pathname}/${col.id}` }
+                            onClick={ event => {
+                                event.preventDefault();
+                                event.stopPropagation();
+                                this.onRowClick(event, col.id);
+                            }}
+                        >
+                            { get(col, path, ' ') || '\u00A0' }
+                        </a>
+                    )) }
+                </div>
                 }
+            );
+        }
 
-                const [ , date, month, year, time ] = new Date(get(col, key, ' ')).toUTCString().split(' ');
-                if (date && month && year && time) {
-                    return `${date} ${month} ${year}, ${time.split(':').slice(0, 2).join(':')} UTC`;
-                }
+        if (isTime) {
+            if (DateFormat) {
```

Comment:
I would probably mark the correct version of frontend components in dependencies or get rid of this. You have basically the same code here twice.

Reply:
This component is hot loaded, meaning that the dep version in package is not used and apps are sending their version. I know it's stupid to check for it, but we don't really have other option in here.

Reply:
Ok